### DISCUSSION TITLE
How are you using Instrumenta?

### DISCUSSION THREAD

**Discussion by iappyx** [iappyx/Instrumenta#5] [Show and tell]
If you like this add-in, please share how you are using Instrumenta in your daily work!

**Comment by Marmootje** [comment]
I definitely like the add-in and use a couple features quite a lot. I am working in investment banking so I work a lot in Powerpoint. I especially like the Select Shapes by Attribute, Align Shapes to Table, Send Selected Slides by E-mail, and Paste and Instert features. This makes it really easy to perform these tasks and speeds up the workflow. 

Many thanks and keep up the good work!

**Comment by davidemora** [comment]
I like it! I collected several VBA macros and created my own addin, but this one is way better and complete. 
Send Selected Slides by E-mail is the feature I am using most.

thanks a lot for this amazing tool!

**Comment by ngito** [comment]
This addin works like a charm, easy installation, and you definitely nailed down most of the productivity related features that took several mouse clicks, this addin reduced it to just a single mouse click.

Feature I used most: swap objects, set object width/height, swap text, remove margins, watermark function, Harvey balls, star ratings, red amber green.

Mail Merge function seems powerful but mysterious, I haven't tried it out.

Thanks for writing this piece of amazing addin!

**Reply by iappyx** [reply]
Thanks! When I have a bit more time, I'll post some tutorials on YouTube for some of the more advanced features like mail merge.

**Comment by Vadigor** [comment]
Randomly came across this while looking to replace the old "Set proofing language" addins and macros I had. Was planning to just reuse your language macro but the add-in does so much more that I wanted to check it out.

Aside from the language setting, the select similar and swap shapes is a lifesaver, the copy&paste position+dimensions has been very useful as well. The "email selected slides" is saving me loads of manual effort too. The stamps are interesting but not as prominent/visible as the ones I typically use so I've avoided them so far in case I forgot to remove them. 😅

The ton of other options will probably have their use as well, looking forward to discovering them.

**Comment by ejazr83** [comment]
I was really amazed to find such a  useful add-in that I can use for my work. The align icons are super useful as are all the fancy options under paste and insert. I am sure there are a lot of other features that I can use but dont know how to.

I was going to make a suggestion if possible to create a how to or user guide that the community can help in developing and just make it easier for newbies like me to fully utilise this great add-in. 

The readme at the moment is quite bare and unless you already know what the icons do, you might not know what other features are missing or how to use them.

**Reply by iappyx** [reply]
Thanks! I am still planning to create a YouTube channel where I can guide users through all the functionality of the tool. Hope to find time for that in Q1 23. The suggestion to create a community based user guide is also very nice. Let me see how that could work on Github. Any suggestions?

**Reply by ejazr83** [reply]
Youtube videos is really nice idea too. Would probably have larger reach.

Community based guide can be on github itself. A lot of projects have a repo for the use guide and people can contribute to it over time. And you can still used pull based requests to approve the contribs

**Comment by ace9790** [comment]
Coming from a consulting background, it was horrible to move into the industry where they dont have 
- good templates
- an own toolbar
- graphics support

As I use ppt on a daily basis, instrumenta allows me to save a lot of time from navigating around the cumbersome standard menus.
I would consider myself a power user, who knows blindly the position of ~50% of all icons.

I also have a few of the non-standard functions added to my quick-access-toolbar, so that I can just press ALT & 1 [or 2, 3, ... ] to instantly perform my most common operations

**Comment by loiroca-xyz** [comment]
This add-in is super useful with its many functions! How can I view the list of shortcuts and potentially edit them?
This would increase my efficiency a lot :)

**Reply by ace9790** [reply]
I left a comment here on that topic :) : https://github.com/iappyx/Instrumenta/issues/37

**Reply by iappyx** [reply]
I have been working on a keyboard shortcut companion called [Instrumenta Keys](https://github.com/iappyx/Instrumenta-Keys). It only works on Windows and is highly experimental.

**Comment by drj-vk** [comment]
I use instrumenta daily in my consulting work. Mostly the note, swap position of two shapes, and reshape. Also the select similar is ideal. Thank you very much for the creation of this wonderful tool!

**Comment by ace9790** [comment]
Just quickly wanted to tell you, @iappyx , again what a rockstar you are!
Thanks for the update! If you ever get bored, I could give you some further improvement options, but those would only take the plugin from 118% amazing to 120% 😆
